recipes/kubernetes-client-python/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: kubernetes
+  version: {{ version }}
+
+source:
+  fn: kubernetes-client-python-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/k/kubernetes/kubernetes-{{ version }}.tar.gz 
+  sha256: 3fa1a50330f0da9d9ba1cd8cc425df481e0326136899875db9448913246f2753
+
+build:
+  # noarch: python
+  preserve_egg_dir: True
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - certifi
+    - ipaddress
+    - oauth2client
+    - setuptools
+    - six
+    - urllib3 !=1.21
+    - python-dateutil
+    - pyyaml
+    - websocket-client
+
+  run:
+    - python
+    - certifi
+    - ipaddress
+    - oauth2client
+    - six
+    - urllib3 !=1.21
+    - python-dateutil
+    - pyyaml
+    - websocket-client
+
+test:
+  imports:
+    - kubernetes
+    - kubernetes.client
+    - kubernetes.client.apis
+    - kubernetes.client.models
+    - kubernetes.config
+    - kubernetes.watch
+  
+about:
+  home: https://github.com/kubernetes-incubator/client-python
+  license: "Apache 2.0"
+  summary: 'The official Kubernetes python client.'
+  license_family: APACHE
```

Comment:
It looks like the upstream source does not define a `MANIFEST.in` so the license isn't packaged with the source. `Apache-2.0` requires that the license be included so please follow:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

and raise and issues/PR upstream to include the license for future releases.